libnet/configure.ac

```diff
@@ -4,25 +4,50 @@ dnl All rights reserved.
 dnl
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT([libnet],[1.2-rc3])
+AC_INIT([libnet],[1.2-rc3],,,[http://github.com/sam-github/libnet])
 AC_MSG_RESULT(beginning autoconfiguration process for libnet-${PACKAGE_VERSION} ...)
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([src/libnet_build_ip.c])
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign subdir-objects no-dependencies -Wall])
 AC_CONFIG_HEADERS([include/config.h])
-AM_MAINTAINER_MODE
 
 dnl
 dnl Check for the usual programs
 dnl
 AC_PROG_CC
 AC_PROG_INSTALL
 AC_USE_SYSTEM_EXTENSIONS
+AX_CFLAGS_WARN_ALL
 dnl disable shared libraries (for now)
 dnl AC_DISABLE_SHARED
+AM_PROG_AR
```

Comment:
> being able to use non-gnu stuff...

Not really, it just disables some checks. From the info pages:
````
‘foreign’
     Automake will check for only those things that are absolutely
     required for proper operations.  For instance, whereas GNU
     standards dictate the existence of a ‘NEWS’ file, it will not be
     required in this mode.  This strictness will also turn off some
     warnings by default (among them, portability warnings).  The name
     comes from the fact that Automake is intended to be used for GNU
     programs; these relaxed rules are not the standard mode of
     operation.
````